### ISSUE TITLE
generating segmentation probability mapping

### ISSUE THREAD

**Issue by charlesyou999648** [xinntao/SFTGAN#9]
Hi Xintao, I am wondering whether it is possible for you to release the code about segmentation probability mapping.

**Comment by xinntao** [comment]
Hi, @charlesyou999648 

We use [Xiaoxiao Li's codes](https://github.com/lxx1991/caffe_mpi) to train our segmentation model and transfer it to a PyTorch model.

It uses a modified Caffe and may take some time to compile and solve the enviroment issues. But it has not updated for a while. 

Maybe try some pytorch repo for segmentation, like [semantic-segmentation-pytorch](https://github.com/CSAILVision/semantic-segmentation-pytorch). It uses Synchronized Batch Normalization and has many pre-trained models.

I am afraid that I do not have enough time to release the training codes for segmentation in the coming several months. 

BTW, [a relevant issue](https://github.com/xinntao/BasicSR/issues/14) in BasicSR.

**Comment by charlesyou999648** [comment]
Thanks a lot!
